receiver/prometheusreceiver/metrics_receiver.go

```diff
@@ -285,6 +304,57 @@ func (r *pReceiver) initPrometheusComponents(ctx context.Context, host component
 			host.ReportFatalError(err)
 		}
 	}()
+
+	// Set up and start the web handler to suppor the server for the Prometheus UI.
+	// Run in agent mode.
+	if r.cfg.EnablePrometheusUIServer {
+		webOptions := web.Options{
+			ScrapeManager: r.scrapeManager,
+			Context:       ctx,
+			ListenAddress: fmt.Sprintf(":%d", prometheusUIServerPort),
```

Comment:
This means only a single prometheus receiver can have the web UI enabled at once, right?